tests/lxd/test_lxd_image_vault.cpp

```diff
@@ -930,3 +914,142 @@ TEST_F(LXDImageVault, minimum_image_size_throws_when_not_found)
                          Property(&std::runtime_error::what,
                                   AllOf(HasSubstr(fmt::format("Cannot retrieve info for image with id \'{}\'", id)))));
 }
+
+TEST_F(LXDImageVault, http_based_image_downloads_and_creates_correct_upload)
+{
+    const std::string content{"This is a fake image!"};
+    mpt::TrackingURLDownloader url_downloader{content};
```

Comment:
Nothing to `EXPECT` on the downloader?

Reply:
Yeah, I think you're right since we do want to verify the downloader was called.  I'll add some expectation.